### PR TITLE
Improve console utility

### LINKED ISSUE #240: Improve migration utility
Need add params, that user should fill in his app (for example. `yii-db-migrations.php` in root folder).

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️

Fix #240

### CODE REVIEW

**Comment by samdark** [comment]
https://github.com/yiisoft/db-migration/blob/master/docs/en/usage-standalone.md and https://github.com/yiisoft/db-migration/blob/master/docs/en/usage-with-symfony.md should be updated.

**Comment by samdark** [comment]
👍 overall.

File: bin/yii-db-migration.bat [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 @setlocal
```
Comment (samdark) [inline bin/yii-db-migration.bat:3]: Let's remove it: https://getcomposer.org/doc/articles/vendor-binaries.md#what-about-windows-and-bat-files-

Comment (vjik) [reply bin/yii-db-migration.bat]: done

**Review by samdark — APPROVED** [review]

**Pull request merged**